pkg/seedadmissioncontroller/webhooks/admission/extensionresources/admission.go

```diff
@@ -105,6 +107,16 @@ func New(logger logr.Logger, allowInvalidExtensionResources bool) *handler {
 			},
 		},
 
+		gvr("etcds"): {
```

Comment:
This function will return the wrong resource group and version, you could create e.g. `gvrDruid` to return the correct ones.